test/lib/sun/hotspot/WhiteBox.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This change isn't in the original patch. Please remove.

Reply:
Odd sorry, I picked the wrong whitebox file!